app/models/User.coffee

```diff
@@ -508,6 +508,10 @@ module.exports = class User extends CocoModel
   skipHeroSelectOnStudentSignUp: -> features?.classroomItems ? false
   useDexecure: -> not (features?.chinaInfra ? false)
   useSocialSignOn: -> not (features?.chinaUx ? false)
+  isTarena: -> features?.Tarena ? false
```

Comment:
Do we need to add it on server also?

Reply:
We just need to add a new feature flag under the `user` type with the name `Tarena`.
The server has a middleware which will be able to correctly send `features.Tarena` for clients created by the Tarena API client. Just need to set it to true for that API client and it'll affect all the users.

Reply:
As far as I know this will be a direct mongo database addition as I don't think we've created a ui to add and remove feature flags.

Reply:
Yeah that sounds right, thanks for clarifying. We may also need to add it to the features collection in database.